test/regression/targets/sc-60301.cc

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file   sc-60301.cc
```

Comment:
Rather than treat this as a regression test for a specific story, I feel like it should be a case in `unit-dense-reader.cc` or the start of a new `unit-dense-global-order-reader.cc`

Apparently there are no global order tests in `unit-dense-reader.cc`?  Yikes.